<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnEnglish</class>
 <widget class="QMainWindow" name="LearnEnglish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LearnEnglish</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>527</width>
        <height>285</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
alternate-background-color: rgb(0, 85, 127);</string>
      </property>
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>200</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/page_2/imgPage_2_2.png);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>250</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>Profil</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>Quitter jeu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>395</x>
          <y>190</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Niveau-1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>395</x>
          <y>215</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Niveau-2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>395</x>
          <y>240</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Niveau-3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Bienvenue  Mr/Mme&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>-24</x>
          <y>-4</y>
          <width>291</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/page_2/imgPage_2_1.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>-30</y>
          <width>291</width>
          <height>351</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/page_2/imgPage_2_1.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>Deconnexion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt; Veillez choisir votre Niveau pour jouer le jeu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <zorder>label_18</zorder>
       <zorder>label_17</zorder>
       <zorder>label_6</zorder>
       <zorder>pushButton_5</zorder>
       <zorder>pushButton_6</zorder>
       <zorder>pushButton_2</zorder>
       <zorder>pushButton_3</zorder>
       <zorder>pushButton_4</zorder>
       <zorder>label_4</zorder>
       <zorder>label_8</zorder>
       <zorder>pushButton_7</zorder>
       <zorder>label_23</zorder>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QListView" name="listView">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Nombre de question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_8">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 255, 127);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Repondre</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_30">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 255);</string>
          </property>
          <property name="text">
           <string>Levels</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Niveau 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Niveau 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Nombre de question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_45">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QListView" name="listView_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_25">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_31">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 255);</string>
          </property>
          <property name="text">
           <string>Levels</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_19">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 255, 127);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Repondre</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0" colspan="3">
         <widget class="QListView" name="listView_6">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_49">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Niveau 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_50">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_48">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Nombre de question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_46">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_33">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 255);</string>
          </property>
          <property name="text">
           <string>Levels</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_32">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 255, 127);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Repondre</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_47">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>listView_6</zorder>
       <zorder>label_49</zorder>
       <zorder>label_48</zorder>
       <zorder>label_50</zorder>
       <zorder>label_47</zorder>
       <zorder>label_46</zorder>
       <zorder>pushButton_33</zorder>
       <zorder>pushButton_32</zorder>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(187, 255, 172);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Niveau suivant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>220</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 127);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Page d'acceuil</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Voulez-vous passé au noveau suivant ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>50</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Félicitation !!!!!!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>-50</x>
          <y>-70</y>
          <width>591</width>
          <height>421</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/page_2/back-ground_2.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_15</zorder>
       <zorder>pushButton_10</zorder>
       <zorder>pushButton_11</zorder>
       <zorder>label_13</zorder>
       <zorder>label_10</zorder>
      </widget>
      <widget class="QWidget" name="page_7">
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 127);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Votre Pseudo:  &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>150</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Votre niveau actuel: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>150</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-image: url(:/page_2/back-ground.png);
image: url(:/page_2/learnEnglishPage_1.png);
font: italic 10pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>481</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(108, 58, 127);
</string>
        </property>
        <property name="text">
         <string>Bienvenue dans votre espace de Jeu d'apprentissage en anglais</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>71</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/page_1/learn-english.png);
background-image: url(:/page_2/back-ground.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="field_pseudo">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>110</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>suivant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>-40</x>
          <y>-10</y>
          <width>611</width>
          <height>351</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/page_2/back-ground.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_16</zorder>
       <zorder>label_2</zorder>
       <zorder>label</zorder>
       <zorder>label_3</zorder>
       <zorder>field_pseudo</zorder>
       <zorder>pushButton</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionA_Propos"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionA_Propos">
   <property name="text">
    <string>A Propos</string>
   </property>
  </action>
  <action name="actionQuitter_Jeux">
   <property name="text">
    <string>Quitter_Jeux</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
